code/game/objects/structures/transit_tubes/transit_tube_construction.dm

```diff
@@ -40,23 +40,27 @@
 	update_icon_state()
 
 /obj/structure/transit_tube_construction/screwdriver_act(mob/living/user, obj/item/I)
-	. = TRUE
```

Comment:
This will cause other shit to break if you remove the trues from here. I suggest keeping these trues as they were before and adding a define bitflag, by putting something like `#define RPD_TOOL_SUCCESS (1<<1)`, and adding `. |= RPD_TOOL_SUCCESS` where you current have your true returns. Then in `automatic_wrench_down` your screwdriver and wrench checks will need to be changed to `if(target.wrench_act(user, src) & RPD_TOOL_SUCCESS)`